zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -580,6 +580,9 @@ import zeppelin from '../../../zeppelin';
 
     $scope.exportToDSV = function(delimiter) {
       var dsv = '';
+      var dateFinished = moment(paragraph.dateFinished).format('YYYY-MM-DD hh:mm:ss A');
+      var exportedFileName = paragraph.title ? paragraph.title + '_' + dateFinished : 'Exported data_' + dateFinished;
```

Comment:
nit: shell we make file name starts with lower case? 
`data_` instead of `Exported data_`, what do you think?